packages/salesforce-adapter/src/filters/layouts.ts

```diff
@@ -99,8 +156,11 @@ const filterCreator: FilterCreator = () => ({
         log.debug('Could not find object %s for layout %s', layoutObjName, layoutName)
         return
       }
+      const layoutObjFields = wu(allCustomObjectFields(elements, layoutObj?.elemID)).toArray()
```

Comment:
when it is just turning an iterable to an array there are built in options, `wu` isn't really required
```suggestion
      const layoutObjFields = [...allCustomObjectFields(elements, layoutObj?.elemID))]
```
mostly a question of style, up to you what you think is best, just making sure you are aware of the options

Reply:
Builtin is better. Thanks!